docs/intro.md

```diff
@@ -47,24 +47,24 @@ Each part goes through the following steps:
 
 The first is that each part is pulled. This step will download
 content, e.g. checkout a git repository or download a binary component
-like the Java SDK. Snapcraft will create a `parts/` directory with
-sub-directories like `parts/part-name/src` for each part that contains
+like the Java SDK. Snapcraft will create a `snap/parts/` directory with
+sub-directories like `snap/parts/part-name/src` for each part that contains
```

Comment:
I'm fine with that as long as we make sure everything is migrated (e.g. I don't think the ROS walkthrough is anywhere. Maybe that belongs on my blog instead?).